src/Cosserat/mapping/DiscreteCosseratMapping.inl

```diff
@@ -688,4 +724,18 @@ void DiscreteCosseratMapping<TIn1, TIn2, TOut>::draw(
   glEnd();
 }
 
+template <class TIn1, class TIn2, class TOut>
+auto DiscreteCosseratMapping<TIn1, TIn2, TOut>::getTildeMatrix(const Vec3 &u)
```

Comment:
Check if it is the same function than `sofa::type::Mat::crossProductMatrix`. If it is, you can remove this function and use `crossProductMatrix` instead.